### PR TITLE
LPS-197569

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
cc: @ccorreagg @ruben-pulido

**Comment by alessandromalves** [comment]
The failures are not related to this PR [(comparison with acceptance routine here)](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1701730356473&testrayRunIdA=2469182737&testrayRunIdB=2470433025). See issues.liferay.com/browse/LPS-166039

**Comment by alessandromalves** [comment]
ci:close

File: modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentResourceTest.java [hunk 1]
```diff
@@ -897,6 +897,53 @@ public void testPutAssetLibraryStructuredContentByExternalReferenceCode()
 					randomStructuredContent3));
 	}
 
+	@Test
+	public void testPutStructuredContentWithDifferentTimeZone()
```
Comment (ccorreagg) [inline modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentResourceTest.java:901]: I think Brian does not like creating new methods in the ResourceTest class. He prefers to override a test, as for example in [this code](https://github.com/liferay/liferay-portal/blob/d3efff982ae783a5f344c3ab977cc8091cad537b/modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentResourceTest.java#L847). Maybe we could do the same with the `testPutStructuredContent` method

**Pull request closed without merging**